packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testGetIconIfAboveThreshold.st

```diff
@@ -0,0 +1,14 @@
+tests
+testGetIconIfAboveThreshold
+
+	| model messageTally expected actual |
+	
+	model := IPTTreeModelMock new. 
+	messageTally := MessageTallyMock new. 
+
+	messageTally tally: 51.
+	model totalTime: 100.
```

Comment:
These method calls can be merged with the instantiation above to improve readability:

```smalltalk
model := IPTTreeModelMock new
    tally: 51.
messageTally := MessageTallyMock new
    totalTime: 100.
```

Or using less lines:

```smalltalk
model := IPTTreeModelMock new tally: 51.
messageTally := MessageTallyMock new totalTime: 100.
```